pkg/api/utils/apiServiceUtils.go

```diff
@@ -52,6 +53,7 @@ func post(uri string, data []byte, api APIService) (*models.EventContext, *model
 				// failed to parse json
 				return nil, buildErrorResponse(err.Error() + "\n" + "-----DETAILS-----" + string(body))
 			}
+			fmt.Println("ID of Keptn context: " + *eventContext.KeptnContext)
```

Comment:
QQ: Is KeptnContext always`!= nil`? I think we should make a sanity check here, right?

Reply:
Thank you, good point! We cannot ensure that the API always returns a valid KeptnContext.